packages/grid/scss/_mixins.scss

```diff
@@ -41,6 +41,13 @@
     padding-right: ($condensed-gutter / 2);
     padding-left: ($condensed-gutter / 2);
   }
+
+  // For our narrow use-case, our container hangs 16px into the gutter
+  .#{$prefix}--row--narrow &,
```

Comment:
```suggestion
  .#{$prefix}--row--narrow,
```

Reply:
Yes I believe so, I followed the same pattern as the condensed grid, these styles need to apply to the children of the class name.